packages/@okta/vuepress-site/docs/reference/api/schemas/index.md

```diff
@@ -2462,3 +2464,351 @@ Specific property types support a subset of [JSON Schema validations](https://to
 | `number`      | [JSON Number](https://tools.ietf.org/html/rfc7159#section-6) with double-precision 64-bit IEEE 754 floating point number constraint | `minimum` and `maximum`     |
 | `integer`     | [JSON Number](https://tools.ietf.org/html/rfc7159#section-6) with 32-bit signed two's complement integer constraint           | `minimum` and `maximum`     |
 | `array`       | [JSON Array](https://tools.ietf.org/html/rfc7159#section-5)                                                                         |                             |
+
+## Unique attribute
+
+<ApiLifecycle access="ea" />
+
+You can enforce uniqueness for custom properties in the Okta user profile or the Okta group profile, such as employee identification number. You can declare a maximum of 5 unique properties for each user type, and 5 unique properties in the Okta group profile. For unique properties in Okta user profile, the limit of 5 unique properties is applied per-type. You do not need to select the same set of properties for each user type. For example, the 5 custom properties you declare unique for user profile A do not need to match what you declared for user profiles B, C, or D.
```

Comment:
```suggestion
You can enforce uniqueness for custom properties in the Okta user profile or the Okta group profile, such as an employee identification number. You can declare a maximum of five unique properties for each user type and five unique properties in the Okta group profile. For unique properties in the Okta user profile, the limit of five unique properties is applied per type. You don't need to select the same set of properties for each user type. For example, the five custom properties that you declare unique for user profile A don't need to match what you declared for user profiles B, C, or D.
```

Reply:
The third sentence is redundant; the previous sentence already says the limit for user properties is "for each user type".  Indeed, I suggest removing all the remaining sentences and condensing them to:
Different user types can have the same or different unique properties (up to the limit of five per type).

I was going to continue with details about interactions between types but I see those have their own paragraphs below, which is probably better.

Reply:
Updated.